prisma/schema.prisma

```diff
@@ -82,3 +84,11 @@ model players {
   highlights                   String?
   users                        users   @relation(fields: [user_id], references: [id])
 }
+
+model viewing_permissions {
+  viewer_id                                  Int     @id
+  viewee_id                                  Int?
+  relationship_type                          String?
+  users_usersToviewing_permissions_viewee_id users?  @relation("usersToviewing_permissions_viewee_id", fields: [viewee_id], references: [id])
+  users_usersToviewing_permissions_viewer_id users   @relation("usersToviewing_permissions_viewer_id", fields: [viewer_id], references: [id])
```

Comment:
Same here, I think this could be better as:
```suggestion
  viewee users?  @relation("UsersToViewees", fields: [viewee_id], references: [id])
  viewer users   @relation("UsersToViewers", fields: [viewer_id], references: [id])
```